plugins/woocommerce/includes/admin/reports/class-wc-report-taxes-by-code.php

```diff
@@ -158,6 +158,8 @@ public function get_main_chart() {
 
 		// Merge.
 		$tax_rows = array();
+		// Initialize an associative array to store unique post_ids.
+		$unique_post_ids = array();
 
 		foreach ( $tax_rows_orders + $tax_rows_partial_refunds as $tax_row ) {
 			$key                                    = $tax_row->rate_id;
```

Comment:
Maybe instead of using `rate_id` here we just want to use `tax_rate`, this will show it more accurately:
![Screenshot 2023-06-01 at 3 47 55 PM](https://github.com/woocommerce/woocommerce/assets/2240960/3bfb275f-836c-4f98-8518-0d937dedb296)

Reply:
Good call @louwie17! I just added the changes you suggested to fix the taxes report. Could you take another look at this PR?